### PR TITLE
kuzco 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by RoseSecurity**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/k/kuzco.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    resource("terraform") do
+      url "https://github.com/hashicorp/terraform/archive/refs/tags/v1.5.7.tar.gz"
+      sha256 "6742fc87cba5e064455393cda12f0e0241c85a7cb2a3558d13289380bb5f26f5"
+    end
+
+    resource("terraform").stage do
+      system "go", "build", *std_go_args(ldflags: "-s -w", output: testpath/"terraform")
+    end
+
+    ENV.prepend_path "PATH", testpath
```
Comment (chenrui333) [inline Formula/k/kuzco.rb:26]: why not use opentofu for testing?

Comment (RoseSecurity) [reply Formula/k/kuzco.rb:26]: Although both are supported, the tool defaults to "terraform," so an additional parameter would be required.

Comment (chenrui333) [reply Formula/k/kuzco.rb:26]: it would be better to use opentofu, as terraform is deprecated in here :)

Comment (RoseSecurity) [reply Formula/k/kuzco.rb:26]: Even pre-BUSL `1.5.7`? I can make the change, just checking!

Comment (chenrui333) [reply Formula/k/kuzco.rb:26]: technically it is fine, but it just makes the test block more cumbersome, and also opentofu would diverge from terraform eventually.

**Comment by chenrui333** [comment]
you need to do the rebase rather than merge to refresh the pr (although no need to care about the outdated base branch)

File: Formula/k/kuzco.rb (hunk 1 above)
Comment (RoseSecurity) [reply Formula/k/kuzco.rb]: Just added opentofu! Thanks for the suggestion

**Pull request closed without merging**